controllers/managedocs_controller.go

```diff
@@ -913,6 +924,67 @@ func (r *ManagedOCSReconciler) reconcileAlertRelabelConfigSecret() error {
 	return nil
 }
 
+func (r *ManagedOCSReconciler) reconcileKubeRbacConfigMap() error {
+	r.Log.Info("Reconciling kubeRbacConfigMap")
+
+	_, err := ctrl.CreateOrUpdate(r.ctx, r.Client, r.kubeRbacConfigMap, func() error {
+		if err := r.own(r.kubeRbacConfigMap); err != nil {
+			return err
+		}
+
+		kubeRbacConfig := struct {
```

Comment:
- maybe use this directly https://github.com/brancz/kube-rbac-proxy/blob/0475166d4bf55981fa704052fd0d2077df6c75f6/pkg/authz/auth.go#L32-L37 and json marshalling?